### PR TITLE
Add-scalafix

### PR DESCRIPTION
**Pull request by Laderlappen**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -31,10 +31,13 @@ jobs:
       - uses: actions/checkout@v4
       - uses: coursier/cache-action@v6
       - uses: VirtusLab/scala-cli-setup@v1
+      - uses: actions/setup-java@v3
         with:
-          jvm: 21
+          distribution: 'zulu'
+          java-version: 21
```
Comment (ff137) [inline .github/workflows/ci.yml:37]: is the extra `setup-java` action necessary? it looks like previously the `jvm: 21` would handle that, without an extra action

Comment (Laderlappen) [reply .github/workflows/ci.yml]: That is correct. I removed the java setup step.

Comment (Laderlappen) [reply .github/workflows/ci.yml]: Removed quotes for the sbt test command.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -31,10 +31,13 @@ jobs:
...
       - run: scala-cli fmt --check .
-      - run: scala-cli --power test .
+      - run: sbt "test"
+      - run: sbt "scalafix --check OrganizeImports"
```
[the remark this answers is not in the document]
Comment (Laderlappen) [reply .github/workflows/ci.yml]: I could remove the quotes in line 39.
But not for line 40.

**Review by ff137 — APPROVED** [review]

**Pull request merged**